.github/actions/vercel/action.yml

```diff
@@ -23,6 +23,7 @@ secrets:
   NEXT_PUBLIC_ETHPASS_KEY:
     required: false
 env:
+  NEXTAUTH_URL: 'https://unlock-protocol.com'
```

Comment:
This is porbabvly not correct?

It needs to be https://app.unlock-protocol.com or https://staging-app.unlock-protocol.com but please pass the value from the workflow that calls this action